bin/MadGraph5_aMCatNLO/patches/mgfixes.patch

```diff
@@ -508,3 +556,16 @@ diff -ur -x '.*/models/.*' ../old/MG5_aMC_v2_3_3/Template/NLO/SubProcesses/MCmas
 +      mcmass(15)=0.0d0
 +      mcmass(16)=0.0d0
        mcmass(21)=0.0d0
+diff -ur ../old/MG5_aMC_v2_3_3/madgraph/interface/madgraph_interface.py MG5_aMC_v2_3_2_2/madgraph/interface/madgraph_interface.py
+--- ../old/MG5_aMC_v2_3_3/madgraph/interface/madgraph_interface.py       2015-09-07 20:07:39.000000000 +0200
++++ MG5_aMC_v2_3_3/madgraph/interface/madgraph_interface.py    2015-11-17 10:50:56.985751642 +0100
+@@ -5851,6 +5851,9 @@
+         elif args[0] in ['cluster_local_path']:
+             self.options[args[0]] = args[1].strip()
+ 
++        elif args[0] in ['cluster_queue']:
```

Comment:
This was the fix for the bug I reported here: https://bugs.launchpad.net/mg5amcnlo/+bug/1517041

The cluster_queue needs to be set to the string "None," but elsewhere the code converts the string to the python None object, so I just added this to keep it as a string.